### PR TITLE
Finally fixes the bird's spelling after 4 years

### PR DESCRIPTION
**Pull request by Iamgoofball**
desktop-green-parrots-pictures

### CODE REVIEW

**Comment by Incoming5643** [comment]
I'm pretty sure that forever ago people did actually call it the singlo. The meme mutated.

**Comment by ShadowDeath6** [comment]
I prefer singlo

**Comment by Mervill** [comment]
if the bird remembers what players say and players always misspell it as singlo when the bird is correct

**Comment by jughu** [comment]
Purge the brain of the bird

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -869,7 +869,7 @@
 /mob/living/simple_animal/parrot/Poly
 	name = "Poly"
 	desc = "Poly the Parrot. An expert on quantum cracker theory."
-	speak = list("Poly wanna cracker!", ":e Check the singlo, you chucklefucks!",":e Wire the solars, you lazy bums!",":e WHO TOOK THE DAMN HARDSUITS?",":e OH GOD ITS FREE CALL THE SHUTTLE")
+	speak = list("Poly wanna cracker!", ":e Check the singulo, you chucklefucks!",":e Wire the solars, you lazy bums!",":e WHO TOOK THE DAMN HARDSUITS?",":e OH GOD ITS FREE CALL THE SHUTTLE")
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/parrot.dm]: It's*

**Comment by dannno** [comment]
its kind of cuter if its singlo

**Pull request merged**